cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3199,6 +3199,27 @@ TEST(TestArrowReaderAdHoc, HandleDictPageOffsetZero) {
   TryReadDataFile(test::get_data_file("dict-page-offset-zero.parquet"));
 }
 
+TEST(TestArrowReaderAdHoc, WriteBatchedNestedNullableStringColumn) {
+  // ARROW-10493
+  auto string_array =
+      ::arrow::ArrayFromJSON(::arrow::utf8(), "[\"a\", \"b\", \"c\", \"d\"]");
```

Comment:
array from json supports structs as {"column_name": value} entries, we could clean this up a little but creating the type first and using ArrayFromJson by itself without use StructArray::Make

Reply:
Nice, thanks!